### ISSUE TITLE
Rate App Button - MVP [Dev-27.0]

### ISSUE THREAD

**Issue by grollk** [ConstructiveNews/squirrel-news-app#77]
User story:
**As an** editor/app owner 
**I want** users to have an easily accessible way to rate the app
**So that** our app rating and number of ratings increases 

Suggested dependency:
https://ionicframework.com/docs/v3/native/app-rate/

Acceptance criteria:
_Given_ the rate app button is clicked
_Then_ a dialog appears asking the user to rate the app

_Given_ the app is opened
_When_ it is the 3rd time 
_Then_ the rate app dialog is opened

_Given_ the rate app dialog is shown
_When_ the user clicks the 'dismiss'-button
_Then_ the dialog closes

_Given_  the rate app dialog is shown
_When_ the user clicks the 'rate'-button
_Then_ the user is redirected to the apps respective store page

_Given_ the rate app dialog is displayed
_Then_ the text and button labels are localized according to the set issue

Dependencies:
none

DoD Checklist:
- [ ] Acceptance criteria are met
- [ ] Test by developer has been done
- [ ] Code is commented if required
- [ ] Code/PR review has been done
- [ ] Feature is documented
- [ ] Internal test by a 2nd Dev or the PO has been done

**Comment by DevJacobi** [comment]
I am currently experimenting with this feature. Currently the prompt always comes when the app is first started, then never again but I configured it otherwise. I need to investigate this more.
Nonetheless: @constructive-news Do you have the texts for the prompt? Maybe I didn't see it in slack.
It currently looks like this in the code
![grafik](https://user-images.githubusercontent.com/43578928/215816111-35d1ef07-1633-4c5f-9ec2-76afa192d873.png)
%@ stands for the App name

**Comment by constructive-news** [comment]
No, we don’t have texts yes. You can use dummy texts for now or just leave it as it is

Jonathan

**Comment by DevJacobi** [comment]
I am not able to get this to work. Either the user is prompted when opening the app first time or never. I give it a few more tries but the time runs out.

**Comment by constructive-news** [comment]
ok, no worries. maybe a simple link can work, too. would be less disturbing anyway

j.
